website/docs/reference/node-selection/methods.md

```diff
@@ -6,6 +6,29 @@ Selector methods return all resources that share a common property, using the
 syntax `method:value`. While it is recommended to explicitly denote the method,
 you can omit it (the default value will be one of `path`, `file` or `fqn`).
 
+<VersionBlock firstVersion="1.5">
+
+:::info New functionality
+New in v1.5!
+:::
+
+Many of the methods below support Unix-style wildcards:
+
+| Wildcard | Description                                               |
+| -------- | --------------------------------------------------------- |
+| \*       | matches any number of any characters (including none)     |
+| ?        | matches any single character                              |
+| [abc]    | matches one character given in the bracket                |
+| [a-z]    | matches one character from the range given in the bracket |
+
+For example:
+```
+dbt list --select **.folder_name.**
+dbt list --select package:*_source
```

Comment:
What does the ** mean?

I needed to add single or double quotes to it to work in my environment:
```suggestion
dbt list --select "*.folder_name.*"
dbt list --select "package:*_source"
```

Without adding quotes, I would get output like this (since the globbing was in my zsh shell rather on the graph nodes):
```
zsh: no matches found: package:*_source
```

Reply:
Great catch!